src/app/app.component.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AppComponent } from './app.component';
+
+describe('AppComponent', () => {
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [
+        RouterTestingModule
+      ],
+      declarations: [
+        AppComponent
+      ],
+    }).compileComponents();
+  });
+
+  it('should create the app', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
+    expect(app).toBeTruthy();
+  });
+
+  it(`should have as title 'survey-challenge'`, () => {
```

Comment:
@carryall Should we keep with `'` format?
So it will be like this `'should have as title \'survey-challenge\''` or `'should have as title "survey-challenge"'`.
As mostly of the block using `'` 🤔
What do you think?

Reply:
Yeah agreed, I'll change it to `'should have as title "survey-challenge"'`

Reply:
✅  fixed in [15742ad](https://github.com/carryall/survey-challenge/pull/1/commits/15742ad3f0d7ac3626d28a6411e9605111e8145e)